banzai/dbs.py

```diff
@@ -293,16 +293,18 @@ class TelescopeMissingException(Exception):
 
```

Comment:
Since `get_telescope_id` can now return `None`,  `populate_bpm_table` might try to `add_or_update_record` with `None` in the `telescope_id` field. Could this be a problem?

Reply:
Yes. We should make sure that doesn't happen. Good catch.

Reply:
Curtis, because we have to do different things to NRES frames and image frames (e.g. trace fitting), why don't we add an attribute to the pipline context, e.g. is_nres_frame? It seems like we will need something like that in the future anyway, and then here we could just have 
```
if pipeline_context.is_nres_frame:
    self.instrument = header.get('TELESCOP')
else:
    self.instrument=  header.get('INSTRUME')
```
 right before the if self.site is not None etc check. Then we could revert back to having get_telescope_id just throwing an exception, and not returning none.

Reply:
I think I would just say, if telescope_id is None, try to set the instrument to Telescop.

Reply:
Ah yes, I agree. that is nice.

Reply:
Is that in the next PR or should I add it into this one?